tests/random_attack.py

```diff
@@ -4,7 +4,7 @@
 #   License, v. 2.0. If a copy of the MPL was not distributed with this
 #   file, You can obtain one at http://mozilla.org/MPL/2.0/.
 #-------------------------------------------------------------------------------
-import sys; sys.path.insert(0, '.'); sys.path.insert(0, '..')
+import sys; sys.path += ('.', '..')
```

Comment:
I don't think this will work: if `.` and `..` are added at the end of the path, then `datatable` module will be found among the installed `site-packages` first, not your working copy.
And btw I don't think `.` is needed either: the lookup is performed in the current directory first by default.

Reply:
I have uninstalled `datatable` prior to start attacks, so that to be sure only a single instance of datatable is tested. However, there is no problem to add `.` and `..` to the beginning of path. Btw, without adding `.` some imports like `from datatable.utils.terminal` won't work, so I will leave it.

Reply:
👍